app/src/main/java/org/kiwix/kiwixmobile/zim_manager/ZimManageActivity.kt

```diff
@@ -128,6 +129,10 @@ class ZimManageActivity : BaseActivity() {
   override fun onOptionsItemSelected(item: MenuItem): Boolean {
     when (item.itemId) {
       R.id.select_language -> start<LanguageActivity>()
+      
+      R.id.get_zim_nearby_device -> {
```

Comment:
this now looks a little awkward next to `R.id.select_language -> start<LanguageActivity>()`
please change to `R.id.select_language -> start<LocalFileTransferActivity>()`

Reply:
[Done](https://github.com/kiwix/kiwix-android/pull/1276/commits/58c936b4fec2f7589cac2249f324c1d74146667a#diff-4bf1f0dbe2d8ffcfc49aa8af0ae42d02R132)
Can `start<ActivityName>()` be used to start any activity in kt?

Reply:
yes, it is an [extension function](https://kotlinlang.org/docs/reference/extensions.html) I created.
To use it in java it would be a much worse `ActivityExtensions.start(activity, ActivityToGoTo.class)`

Kotlin is a lot more powerful than java, hopefully we get you writing some soon

Reply:
I would like to start writing kotlin code by the end of this year, since it seems like all apps will go kotlin-first soon enough, and so I'm way behind schedule
But I also have to prepare my master's thesis and get a job this academic year, so it's also not at the top of my priority list right now